app/src/test/java/com/shagalalab/constitution/ui/article/ArticleViewModelTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.shagalalab.constitution.ui.article
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.shagalalab.constitution.TestExecutor
+import com.shagalalab.constitution.data.dao.ArticleDao
+import com.shagalalab.constitution.data.models.ArticleModel
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class ArticleViewModelTest {
+
+    private lateinit var viewModel: ArticleViewModel
+    private val articleModel = ArticleModel(1)
+    private val articleModels = listOf(articleModel)
+    private val executor = TestExecutor()
+    private val articleMockDao = mockk<ArticleDao>()
+
+    @Before
+    fun setUp() {
+        viewModel = ArticleViewModel(articleMockDao, executor)
+    }
+
+    @get: Rule
+    val rule = InstantTaskExecutorRule()
+
+    @Test
+    fun getArticlesByChapterId() {
```

Comment:
Let's be consistent, I proposed `verify abcMethod <does xyz>` as a template for method naming, it is better to stick to that.